### PR TITLE
Migrate to `main` branch

### LINKED ISSUE #3686: Migrate from `master` to `main` (branch)
When possible

https://dev.to/tham/how-to-migrate-master-branch-to-main-in-github-48aj

Update settings (branch protection, CI checks, etc.)

### PR DESCRIPTION
**Pull request by julieg18**
**What this pr does:**
* update package.json commands
* update github actions
* update circleci config
* update `getEditLink.ts`

**What we need to do after this is merged:**
* Rename `master` branch
* [Update our heroku pipeline](https://help.heroku.com/O0EXQZTA/how-do-i-switch-branches-from-master-to-main)
* Makes sure our gh actions and circle ci area all working

Fixes #3686

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,10 +22,10 @@ commands:
       # CircleCI breaks master branch which affects format check below. See
```
Comment (julieg18) [inline .circleci/config.yml:22]: I updated `config.yml` but is there anything else that needs to be done for `circleci` to be run correctly?

File: src/@dvcorg/gatsby-theme-iterative/getEditLink.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
 const getEditLink = (sourcePath: string): string =>
-  `https://github.com/iterative/dvc.org/blob/master/content${sourcePath}`
```
Comment (julieg18) [inline src/@dvcorg/gatsby-theme-iterative/getEditLink.ts:2]: Is there anything else I need to do besides what I've mentioned in the pr?

**Review by yathomasi — COMMENTED** [review]
I think they are the ones on the website. But, there are on the content side as well. We should be updating them as well. 

dvc.org/content/docs/contributing/docs.md (Line 193)
Screen Shot 2022-06-23 at 18 06 11

File: .circleci/config.yml (hunk 1 above)
Comment (rogermparent) [reply .circleci/config.yml:22]: I'm looking at the CircleCI settings ui and don't see anything related to branches, so I think we're good there!

**Comment by julieg18** [comment]
> I think they are the ones on the website. But, there are on the content side as well. We should be updating them as well.

Nice catch! Code is updated!

File: content/docs/contributing/docs.md [hunk 3]
```diff
@@ -155,7 +155,7 @@ that is installed when `yarn` runs (see [dev env](#development-environment)).
   > plugin for help with this.
 
 - You can see the configuration of our formatter tool (Prettier)
-  [here](https://github.com/iterative/dvc.org/blob/master/.prettierrc). You may
+  [here](https://github.com/iterative/dvc.org/blob/main/.prettierrc). You may
```
Comment (yathomasi) [inline content/docs/contributing/docs.md:158]: There is one left in Line 193.

**Comment by rogermparent** [comment]
I see a couple remaining from a ripgrep:
```
rg "dvc.org.*master"
content/blog/2020-04-30-gsod-ideas-2020.md
59:[dozens of tickets](https://github.com/iterative/dvc.org/pulls?q=is%3Apr+author%3Aalgomaster99+is%3Aclosed)
215:   [source code](https://github.com/iterative/dvc.org/tree/master/src) we've

content/docs/contributing/docs.md
20:- [Content](https://github.com/iterative/dvc.org/tree/master/content/docs)
24:- [Images](https://github.com/iterative/dvc.org/tree/master/static/img)
27:- [Navigation](https://github.com/iterative/dvc.org/tree/master/content/docs/sidebar.json)
193:> [this very file](https://raw.githubusercontent.com/iterative/dvc.org/master/content/docs/user-guide/contributing/docs.md).
```

**Comment by rogermparent** [comment]
Also in `contributing/docs`
```
Merging the appropriate changes to these files into the master branch is enough
to update the docs and redeploy the website.
```

File: content/docs/contributing/docs.md [hunk 4]
```diff
@@ -17,19 +17,19 @@ and then **Commit changes** from the bottom of the page.
...
   (`static/img/`): Add new images (`.png`, `.svg`, etc.) here. Use them in
   Markdown files like this: `![](/img/<filename>.gif)`.
-- [Navigation](https://github.com/iterative/dvc.org/tree/master/content/docs/sidebar.json)
+- [Navigation](https://github.com/iterative/dvc.org/tree/main/content/docs/sidebar.json)
   (`content/docs/sidebar.json`): Edit it to add or change entries in the
   navigation sidebar.
 
-Merging the appropriate changes to these files into the master branch is enough
-to update the docs and redeploy the website.
+Merging the appropriate changes to these files into the main branch is enough to
```
Comment (jorgeorpinel) [inline content/docs/contributing/docs.md:31]:
```suggestion
Merging the appropriate changes to these files into the `main` branch is enough to
```

**Review by jorgeorpinel — APPROVED** [review]
LGTM. There's no need to update the blog posts (GH will redirect from master URL to main anyway I think) but doesn't hurt either so up to you.

**Comment by julieg18** [comment]
Renaming (then merging) now!

**Pull request merged**